src/user/using-the-capture-app.md

```diff
@@ -887,3 +887,9 @@ This is the form where you can see and edit the enrollment event details.
 In the view/edit event form you can see the stage name and icon.
 
 ![](resources/images/capture_app/enrollment-event-view-edit-header.png)
+
+#### Widgets in View/Edit event page
+
+The widgets seen in the right-hand column will display and function the same way as mentioned in the enrollment dashboard.
```

Comment:
I know that the section being referenced here is just above this one, but for someone landing on this section from another place it may be useful to link the section you are referencing.

```suggestion
The widgets seen in the right-hand column will display and function the same way as mentioned in the [enrollment dashboard](#enrollment-dashboard).
```